### ISSUE TITLE
PSR-7 UploadedFileInterface support

### ISSUE THREAD

**Issue by realrecordzLab** [verot/class.upload.php#143]
Is this library support PSR-7 UploadedFileInterface ? I want to integrate it with slim framework 4 that is using that interface when a file is uploaded

**Comment by verot** [comment]
No, the library doesn't support UploadedFileInterface. But I will gladly merge a patch.

**Comment by realrecordzLab** [comment]
Ok, it will be a nice feature to make the library framework agnostic. If
you give me more details about the code to modify I can try to make a patch

Il giorno sab 14 nov 2020 alle ore 22:04 Colin Verot <
notifications@github.com> ha scritto:
